doc/guide/Guide_PythonDos-n-Donts.ipynb

```diff
@@ -570,7 +570,14 @@
     "    return 'hello there'\n",
     " </code></pre>\n",
     " \n",
-    "* type checking (Python is a dynamically typed language; also: cf. \"Duck typing\". Yet, as a best practice, variables should not change type once assigned, and mutable default parameters are banned) "
+    "* type checking (Python is a dynamically typed language; also: cf. \"Duck typing\". Yet, as a best practice, variables should not change type once assigned)\n",
+    "* Do not use mutable default arguments in your functions (e.g. lists). For example, if you define a function as such:\n",
+    "\n",
+    "    ```\n",
+    "    def function(x, list=[]):    \n",
+    "        default_list.append(x)\n",
+    "    ```\n",
+    "   your list will be mutated for future calls of the functions too."
```

Comment:
Could you add the correct implementation to complete the example? Something like
```
def func(x, list=None):
    list = [] if list is None
```

Reply:
https://github.com/CLIMADA-project/climada_python/pull/181/commits/342b060b49d893f0aa306b7a222f59bdb065d4a3